package.json

```diff
@@ -29,7 +29,9 @@
     "ea-start": "./build.js -run -watch --settings ../ForumCredentials/settings-dev.json --mongoUrlFile ../ForumCredentials/dev-db-conn.txt",
     "ea-start-staging-db": "./build.js -run --watch --mongoUrlFile ../ForumCredentials/staging-db-conn.txt --settings ../ForumCredentials/settings-local-dev-staging-db.json",
     "ea-start-prod-db": "./build.js -run --watch --mongoUrlFile ../ForumCredentials/prod-db-conn.txt --settings ../ForumCredentials/settings-local-dev-prod-db.json",
-    "ea-start-memory-db": "./scripts/mongoMemoryServer.js & ./build.js -run --watch --mongoUrl mongodb://localhost:27017 --settings ./cypress/instance-settings-test.json",
```

Comment:
Given that you're removing this, shouldn't you be removing the `./scripts/mongoMemoryServer`? Or do you want to keep that one for the build.js case where there's no mongodb specified?